Backend/Controllers/AuthController.cs

```diff
@@ -26,8 +24,10 @@ public static void MapAuthEndpoints(this WebApplication app)
                     var loggedInUser = userService.GetUserByUsername(credentials.Username);
                     var claims = new[]
                     {
-                        new Claim(JwtRegisteredClaimNames.Sub, user.Username),
-                        new Claim(ClaimTypes.Role, loggedInUser.Role)
+                        new Claim(JwtClaimNames.Sub, loggedInUser.Username),
+                        new Claim("role", loggedInUser.Role),
+                        new Claim(ClaimTypes.Role, loggedInUser.Role),
+                        new Claim(JwtClaimNames.Jti, Guid.NewGuid().ToString())
                     };
 
                     var key = new SymmetricSecurityKey(Encoding.UTF8.GetBytes("UngnjU6otFg8IumrmGgl-MbWUUc9wMk0HR37M-VYs6s="));
```

Comment:
Eh, sec risk?

Reply:
Ye, men jeg hardcoder den også i backenden desværre. Skal vi huske at fjerne før eksamen, skriver det ind i vores security dokument.

Reply:
Plus at det er noget der er kommet ind i et tidligere pr. Men måske en fair tanke at få lavet en opgave på at det skal ud, det kan jeg lige gøre

Reply:
https://trello.com/c/YXYurZDh/70-update-key-encoding-not-to-be-present-in-backend